.github/workflows/ci-publish-base-image.yml

```diff
@@ -1,6 +1,8 @@
 name: Publish Base CI Image To Dockerhub
 
 on:
+  create:
+    branches: [main, release-*, gha-test-*]
```

Comment:
I think only release-* is really needed here which needs bootstrap .

Reply:
Probably should include it for everything but auto/canary since ci-test.yml needs a base image for the repo name.   If anything we should expand this.

Reply:
almost spent time on using branches-ignore but don't want to waste time looking to see how it would handle some of the other refs hanging of the repo.   maybe in the future.

```
on:
  create:
    branches-ignore: [auto, canary]
```